code/game/objects/items/storage/belt.dm

```diff
@@ -136,6 +135,43 @@
 	new /obj/item/tool/wirecutters(src)
 	new /obj/item/device/t_scanner(src)
 
+/obj/item/storage/belt/utility/construction
+	name = "\improper M277 pattern construction rig"
+	desc = "The M277 is a common rig used by Combat Technicians to carry around materials and other supplies. It consists of a modular belt with various clips. This version sarafices storage space for specialized material loading clips."
+	storage_slots = 6
+	can_hold = list(
```

Comment:
Barbed wire and sandbags aren't included but metal and plasteel is.

Reply:
Fixed